CBinding/Generated.cs

```diff
@@ -0,0 +1,2055 @@
+    {
+        public CXFile(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    public partial struct CXDiagnostic
+    {
+        public CXDiagnostic(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    public partial struct CXDiagnosticSet
+    {
+        public CXDiagnosticSet(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    public partial struct CXCursorSet
+    {
+        public CXCursorSet(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+    public delegate CXChildVisitResult CXCursorVisitor(CXCursor @cursor, CXCursor @parent, IntPtr @client_data);
+
+    public partial struct CXModule
+    {
+        public CXModule(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+	[StructLayout (LayoutKind.Sequential)]
+    public partial struct CXCompletionString
+    {
+        public CXCompletionString(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+    public delegate void CXInclusionVisitor(IntPtr @included_file, out CXSourceLocation @inclusion_stack, uint @include_len, IntPtr @client_data);
+
+    public partial struct CXRemapping
+    {
+        public CXRemapping(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    public partial struct CXIdxClientFile
+    {
+        public CXIdxClientFile(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    public partial struct CXIdxClientEntity
+    {
+        public CXIdxClientEntity(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    public partial struct CXIdxClientContainer
+    {
+        public CXIdxClientContainer(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    public partial struct CXIdxClientASTFile
+    {
+        public CXIdxClientASTFile(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    public partial struct CXIndexAction
+    {
+        public CXIndexAction(IntPtr pointer)
+        {
+            this.Pointer = pointer;
+        }
+
+        public IntPtr Pointer;
+    }
+
+    [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+    public delegate CXVisitorResult CXFieldVisitor(CXCursor @C, IntPtr @client_data);
+
+    public enum CXErrorCode : uint
+    {
+        @CXError_Success = 0,
```

Comment:
I know it's not your code but it would be nice to strip the prefix from these enums so it becomes e.g. `CXErrorCode.Success` instead of `CXErrorCode.CXError_Success`. Should be a pretty easy change in the generator. Not a big deal though, could be done later.